<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>PathToolOptionWidgetBase</class>
 <widget class="QWidget" name="PathToolOptionWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>208</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wdgShapeProperties" native="true"/>
   </item>
   <item>
    <widget class="Line" name="lineShapeProperties">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pathPage">
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item column="0" row="0">
        <widget class="QToolButton" name="corner">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="1" row="0">
        <widget class="QToolButton" name="smooth">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="2" row="0">
        <widget class="QToolButton" name="symmetric">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="3" row="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item column="5" row="0">
        <widget class="QToolButton" name="lineSegment">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="6" row="0">
        <widget class="QToolButton" name="curveSegment">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="7" row="0">
        <widget class="QToolButton" name="linePoint">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="8" row="0">
        <widget class="QToolButton" name="curvePoint">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="0" row="1">
        <widget class="QToolButton" name="addPoint">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="1" row="1">
        <widget class="QToolButton" name="removePoint">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="5" row="1">
        <widget class="QToolButton" name="breakPoint">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="6" row="1">
        <widget class="QToolButton" name="breakSegment">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="7" row="1">
        <widget class="QToolButton" name="joinSegment">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="8" row="1">
        <widget class="QToolButton" name="mergePoints">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item column="4" row="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item column="4" row="1">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item column="5" row="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="parameterPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="convertToPath">
           <property name="text">
            <string>Convert to Path</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>150</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>